rules/type-declaration/src/TypeInferer/PropertyTypeInferer/DoctrineRelationPropertyTypeInferer.php

```diff
@@ -88,6 +89,7 @@ private function processToOneRelation(
 
         $fullyQualifiedTargetEntity = $toOneTagNode->getFullyQualifiedTargetEntity();
```

Comment:
I think method should return class name without `::class`, as it's in the name.
Why not use it?

Reply:
https://github.com/rectorphp/rector/pull/3770/commits/24d7b66a87c6adf5b9b8c23ecc03998eaa87dc59#diff-5641698a38562059eecf39ef68b4b9efL62-R62

What about this place to cleanup the FQCN?

Reply:
Great!